optimum/habana/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py

```diff
@@ -34,16 +38,714 @@
     CLIPVisionModelWithProjection,
 )
 
+from optimum.habana.diffusers import GaudiDiffusionPipeline
+from optimum.habana.diffusers.models.attention_processor import (
+    AttentionProcessor,
+    AttnProcessor2_0,
+    ScaledDotProductAttention,
+)
+from optimum.habana.diffusers.models.unet_2d_condition import gaudi_unet_2d_condition_model_forward
+from optimum.habana.diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion import (
+    retrieve_timesteps,
+)
+from optimum.habana.transformers.gaudi_configuration import GaudiConfig
+from optimum.habana.utils import HabanaProfile, speed_metrics, warmup_inference_steps_time_adjustment
```

Comment:
```suggestion
from ...diffusers import GaudiDiffusionPipeline
from ...diffusers.models.attention_processor import (
    AttentionProcessor,
    AttnProcessor2_0,
    ScaledDotProductAttention,
)
from ...models.unet_2d_condition import gaudi_unet_2d_condition_model_forward
from ..pipelines.stable_diffusion.pipeline_stable_diffusion import (
    retrieve_timesteps,
)
from ....transformers.gaudi_configuration import GaudiConfig
from ....utils import HabanaProfile, speed_metrics, warmup_inference_steps_time_adjustment
```
Please use relative imports in the source code